File: packaging/linux/runtime-sdk/java/src/main/java/io/aegis/runtime/AegisRuntime.java
package io.aegis.runtime;

public final class AegisRuntime {
    public static final String CONTRACT_VERSION = "serverless.v1";

    private AegisRuntime() {}

    public static String buildSummary() {
        return "{\"language\":\"java\",\"signal_kind\":\"HttpRequest\",\"connector_id\":\"aws-cloudtrail\"}";
    }
}

File: packaging/linux/runtime-sdk/dotnet/AegisRuntime.cs
namespace Aegis.Runtime;

public static class AegisRuntime
{
    public const string ContractVersion = "serverless.v1";

    public static string BuildSummary()
    {
        return "{\"language\":\"dotnet\",\"signal_kind\":\"HttpRequest\",\"connector_id\":\"azure-activity\"}";
    }
}
